e2e-tests/cypress/e2e/journeys/insurance/your-business/nature-of-business/validation/nature-of-business-years-exporting-validation.spec.js

```diff
@@ -141,6 +149,8 @@ describe('Insurance - Your business - Nature of your business page - As an Expor
       const field = natureOfBusiness[fieldId];
 
       completeField(GOODS_OR_SERVICES, 'abc');
+      completeField(EMPLOYEES_UK, '5');
+      completeField(EMPLOYEES_INTERNATIONAL, '5');
```

Comment:
These additions seem troublesome to maintain (same in previous file). Instead of having to add new data entry to each test like this (in order to test a single error message) could we not just do e.g `partials.errorSummaryListItems().eq(3),` instead? Did this in policy and exports and worked well

Reply:
should I change in this PR?

Reply:
I think so, don't think it'd take long? Otherwise we're introducing tech debt?